spec/rubocop/cop/lint/format_parameter_mismatch_spec.rb

```diff
@@ -138,6 +138,24 @@
     expect(cop.offenses).to be_empty
   end
 
+  it 'does not register an offense on argument splat' do
```

Comment:
This description could be better. Now it reads like it never registers offences if a splat arg is present.